### PR TITLE
Search page template

### PR DESCRIPTION
**Pull request by finnhodgkin**
Adds search html and book component #11

### CODE REVIEW

File: assets/css/app.css [hunk 1]
```diff
@@ -1 +1,44 @@
...
+.f-dark-grey{
+  fill: #4D4D4D;
+}
+
+.bg-light-grey{
+  background-color: #EBEBEB;
+}
+
+.bs-bb{
+  box-sizing: border-box;
```
Comment (katbow) [inline assets/css/app.css]: Isn't there a `.border-box` with tachyons already?

File: lib/library_web/templates/layout/app.html.eex [hunk 2]
```diff
@@ -8,17 +8,29 @@
...
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.6.1/css/tachyons.min.css"/>
     <link rel="stylesheet" href="<%= static_path(@conn, "/css/app.css") %>">
   </head>
 
-  <body>
-    <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
-    <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
+  <body class="bg-light-grey">
+    <div class="h3 pb1 pt2">
+      <img src="/images/dwyl-heart-only-logo-white.png" class="w3-5 mt1 db center"/>
```
Comment (katbow) [inline lib/library_web/templates/layout/app.html.eex]: There's the [phoenix `img_tag`](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Tag.html#img_tag/2) we should use for images.

File: lib/library_web/templates/layout/app.html.eex [hunk 3]
```diff
@@ -8,17 +8,29 @@
...
+    </div>
+    <nav class="h3 pb1 pt2 fixed w-100 bg-dwyl-teal top-0">
+      <img src="/images/dwyl-heart-only-logo-white.png" class="w3-5 mt1 db center"/>
+      <div class="right-0 top-0 absolute pt3">
+        <a href="/search" class="link db white fr f5 mr3 mt1 mb1">Search</a>
+        <a href="/feedback class" class="link db white right f5 mr3">Feedback</a>
```
Comment (katbow) [inline lib/library_web/templates/layout/app.html.eex]: There's also a [phoenix `link`](https://hexdocs.pm/phoenix_html/Phoenix.HTML.Tag.html#img_tag/2) which can be used with the [path helpers](https://hexdocs.pm/phoenix/routing.html#path-helpers).

File: lib/library_web/templates/layout/app.html.eex [hunk 4]
```diff
@@ -8,17 +8,29 @@
...
+      </div>
+    </nav>
 
     <main role="main">
       <%= render @view_module, @view_template, assigns %>
     </main>
 
+    <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
+    <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
```
Comment (katbow) [inline lib/library_web/templates/layout/app.html.eex]: Might want to change the styling of these now with tachyons, but could also wait until we know what we want these to look like. But it's nice to get rid of any bootstrap remnants soon as!

**Comment by finnhodgkin** [comment]
I think I've fixed everything apart from border-box because tachyons doesn't reset everything or include a border-box class :cry:

**Comment by katbow** [comment]
Merging. Can we have the README update in the next pr please?

**Pull request merged**